supervisor/host/configuration.py

```diff
@@ -340,3 +353,14 @@ def _map_nm_vlan(inet: NetworkInterface) -> VlanConfig | None:
             return None
 
         return VlanConfig(inet.settings.vlan.id, inet.settings.vlan.parent)
+
+    @staticmethod
+    def _map_nm_multicast_dns(mode: int | None) -> MulticastDnsMode | None:
+        mapping = {
+            MulticastDnsValue.DEFAULT: MulticastDnsMode.DEFAULT,
+            MulticastDnsValue.OFF: MulticastDnsMode.OFF,
+            MulticastDnsValue.RESOLVE: MulticastDnsMode.RESOLVE,
+            MulticastDnsValue.ANNOUNCE: MulticastDnsMode.ANNOUNCE,
```

Comment:
```suggestion
            MulticastDnsValue.DEFAULT.value: MulticastDnsMode.DEFAULT,
            MulticastDnsValue.OFF.value: MulticastDnsMode.OFF,
            MulticastDnsValue.RESOLVE.value: MulticastDnsMode.RESOLVE,
            MulticastDnsValue.ANNOUNCE.value: MulticastDnsMode.ANNOUNCE,
```

Reply:
tbh i would also extract this mapping into constant to avoid runtime allocation overhead, although i am not a Python-expert at all, so it can be just a worthless micro-optimization